installer/datafiles/base_omsagent.data

```diff
@@ -25,6 +25,8 @@ MAINTAINER:              'Microsoft Corporation'
 /etc/opt/microsoft/omsagent/sysconf/omsagent.d/change_tracking.conf;    installer/conf/omsagent.d/change_tracking.conf;        644; root; root
 /etc/opt/microsoft/omsagent/sysconf/omsagent.d/change_tracking_inventory.mof;    installer/conf/omsagent.d/change_tracking_inventory.mof;    644; root; root
 /etc/opt/microsoft/omsagent/sysconf/omsagent.d/monitor.conf;            installer/conf/omsagent.d/monitor.conf;                644; root; root
+/etc/opt/microsoft/omsagent/sysconf/omsagent.d/patch_management.conf;      installer/conf/omsagent.d/patch_management.conf;          644; root; root
+/etc/opt/microsoft/omsagent/sysconf/omsagent.d/patch_management_inventory.mof;  installer/conf/omsagent.d/patch_management_inventory.mof;        644; root; root
```

Comment:
Our current plan for private preview was to let customers install the bundle, and then run the following commands and restart the agent.

`sudo cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/patch_management.conf /etc/opt/microsoft/omsagent/conf/omsagent.d/`
`sudo cp /etc/opt/microsoft/omsagent/sysconf/omsagent.d/patch_management_inventory.mof /etc/opt/microsoft/omsagent/conf/omsagent.d/`

This was later to be updated by DSC Channel.